README.md

```diff
@@ -65,6 +65,10 @@ This action installs packages required for proper `obal` usage on your machine.
 
 This action updates a package to a newer version. The newer version is either taken from an upstream repository or can be given via command line options
 
+### bump-release
+
+This action updates a spec file's release field and adds a pre-formatted changelog entry.
```

Comment:
This makes me wonder if we should somehow include this in `obal --help` (but out of scope for this PR).